src/Components/Display.js

```diff
@@ -0,0 +1,43 @@
+import React,{useState} from 'react';
+import axios from 'axios';
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import "./display.css"
+
+const Display = (props) => {
+    const [state,setState]=useState("")
+    const navigate=useNavigate();
+    
+    useEffect(() => {
+        axios.get('http://localhost:5000/display/'+props.email)
+        .then(response => {
+            setState(response.data);
+            console.log(response.data)
+        })
+        .catch(function (error){
+            console.log(error);
+        })
+    }, [0])
+    return (
+    <div className='display-data'>
+        
+        Your details:<br></br>
+        
+        First Name:{state.firstname}<br></br>
```

Comment:
This would display as "First Name:Vignesh" which doesn't look good. Please change it to "First Name: Vignesh"

Reply:
Follow the same for all of the details below